test/e2e/specs/testdriver.go

```diff
@@ -66,6 +67,7 @@ type gcsVolume struct {
 // InitGCSFuseCSITestDriver returns GCSFuseCSITestDriver that implements TestDriver interface.
 func InitGCSFuseCSITestDriver(c clientset.Interface, m metadata.Service, bl string, skipGcpSaTest, enableHierarchicalNamespace bool, clientProtocol string, enableZB bool) storageframework.TestDriver {
 	ssm, err := storage.NewGCSServiceManager()
+	storageService, err := ssm.SetupServiceWithDefaultCredential(context.Background(), enableZB)
```

Comment:
Where does SetupServiceWithDefaultCredential get called right now for e2e test runs? Trying to avoid calling it again here.